docs/section9/Rule6-w.md

```diff
@@ -1,44 +1,37 @@
 
-# Lighting - Rule 6-w
+# Lighting - Rule 6-12
 
-**Rule ID:** 6-w  
-**Rule Description:** Proposed building is modeled with dayligthing controls and no daylighting control is modeled for baseline  
-**Appendix G Section:** Lighting  
-**Appendix G Section Reference:** None  
+**Rule ID:** 6-12  
+**Rule Description:** Proposed building is modeled with dayligthing controls  
+**Appendix G Section:** Section G3.1-6(h) Lighting: Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:**  
 
-**Applicability:** All required data elements exist for P_RMR and B_RMR  
+**Applicability:** All required data elements exist for P_RMR  
 **Applicability Checks:** None  
 **Manual Check:** Yes  
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
 ## Rule Logic: 
 
-- Check if each space has window or skylight in the building segment in the Proposed model: ```For building_segment in P_RMR.building.building_segments:```  
+- Check if each space has window or skylight in the building segment in the Proposed model: ```For building_segment_p in P_RMR.building.building_segments:```  
 
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - For each thermal_block in building segment: ```thermal_block_p in building_segment_p.thermal_blocks:```  
 
-  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+  - For each zone in thermal block: ```zone_p in thermal_block_p.zones:```  
 
-  - Get space from thermal zone: ```space in thermal_zone.spaces:```  
+    - Get surfaces in zone: ```surface_p in zone_p.surfaces```  
 
```

Comment:
@mkarpman with surfaces now assigned to zones, the daylighting check will happen at the zone level and not at the space ;level. Please review the rule assertions.